storage/byodb/external-buckets/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: External Buckets
+permalink: /storage/byodb/external-buckets/
+---
+
+* TOC
+{:toc}
+
+If you operate Keboola Connection in Bring Your Own Database (BYODB) mode atop your own data warehouse, the data residing in the warehouse will not automatically be visible or accessible from inside Keboola Connection. 
+We address this by providing the **External Buckets** feature.
+
+The exact implementation of external buckets depends on the database backend you use; it could be a database, schema, or another concept. Unless stated otherwise, we will describe the implementation for Snowflake.
+
+## What Is an External Bucket?
+
+Storage in Keboola Connection is organized into [buckets](/storage/buckets/). An **external bucket** is a special type of bucket wherein the contents of the bucket are not managed by Keboola. 
+It can be located anywhere in your data warehouse and is a virtual bucket created atop a Snowflake schema. All table-like objects (such as tables, views, and external tables) inside the schema 
```

Comment:
`All table-like objects (such as tables, views, and external tables)` tohle neni uplne pravda, ale asi to nevadi. vlastne nam chybi views a external tables v BQ.

Reply:
a teda `in your data warehouse which is attached to your Keboola project` aby bylo jasne, ze perimetr veci co muzes nalinkovat do kebooly skrze EB je omezene warehousem, nad kterym bezi tvuj projekt. Proste bych rad, aby zde bylo explicitne receno, ze kdyz mas jinej snflk na druhy strane zemekoule, tak na to EB nejsou urceny